src/thunderbots/shared/constants.h

```diff
@@ -11,7 +11,8 @@ const double BALL_MAX_SPEED_METERS_PER_SECOND = 6.5;
 const double ROBOT_MAX_RADIUS_METERS = 0.09;
 // The approximate radius of the ball according to the SSL rulebook
 const double BALL_MAX_RADIUS_METERS = 0.0215;
-
+// The maximum number of robots we can communicate with.
+const unsigned MAX_ROBOTS = 8;
```

Comment:
Could we rename this something more radio-specific since (if I understand this correctly) it's a radio constraint. Maybe something like `MAX_ROBOTS_OVER_RADIO`?

Reply:
changed